glib/src/gobject/signal_group.rs

```diff
@@ -42,6 +42,44 @@ impl SignalGroup {
         self.connect_closure(signal_name, after, RustClosure::new_local(callback));
     }
 
+    #[inline]
+    pub fn connect_notify<F>(&self, name: Option<&str>, callback: F)
```

Comment:
Why is this needed? It should be possibly to directly call http://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib/object/trait.ObjectExt.html#tymethod.connect_notify on the `SignalGroup` as it's a `glib::Object`

Reply:
Because of the nature of `SignalGroup`.

`SignalGroup` is an object that makes it easy to listen to multiple signals from the `target` object. This means that the `SignalGroup::connect` will listen for signals of the `target` object, instead of the `SignalGroup` itself. This PR adds an helper to connect to notify signals of properties of the `target` object, which is different than `ObjectExt::connect_notify`, because the latter only listens to notify signals of the `SignalGroup` object, which is not what we want.